pkg/console/controllers/route/controller.go

```diff
@@ -174,7 +174,16 @@ func (c *RouteSyncController) removeRoute(routeName string) error {
 }
 
 func (c *RouteSyncController) SyncDefaultRoute(operatorConfig *operatorsv1.Console) (*routev1.Route, string, error) {
-	requiredDefaultRoute := routesub.DefaultRoute(operatorConfig)
+	customTLSSecret, configErr := c.ValidateDefaultRouteConfig(operatorConfig)
+	if configErr != nil {
+		return nil, "InvalidDefaultRouteConfig", configErr
```

Comment:
It's not necessarily invalid. It means we couldn't get the secret, right? It be that the name is wrong, but it could be due to other errors as well?